docs/design/chart_format.md

```diff
@@ -189,27 +164,16 @@ keywords:
 * proxy
 source: https://github.com/foo/bar
 home: http://nginx.com
-depends:
-        charts:
-          - name: memcached
-                location: helm.sh/charts/memcached
-                version: 0.9.3
-        kubernetes:
-                version: “>=1.0.0”
-                extensions:
-                  - extensions/v1beta1
+expander:
+  name: goexpander-service
+schema: Schema.yaml
 ```
 
-#### Dependency Information
-The chart format allows the specification of _runtime dependencies_ (not build-time dependencies). For example, a Wordpress chart might have a runtime dependency on a MySQL database. The database needn’t be present for processing the template file, but must be present in the cluster in order for Wordpress to function.
-
-
-Dependencies hold between charts. That is, one chart may depend upon another chart.
-
+### Expanders and Templates
+The content of the `templates/` directory and the schema file are defined by the particular expander invoked by name in the Chart.yaml file.  Such expanders consume these files, in the context of properties, to generate content.  If a schema is given, the expander may use it to validate the properties before expansion.  Discussion of the available expanders and how they intepret the content of /templates is outside the scope of this document.
```

Comment:
Add...

If no expander is specified, files with `yaml` or `json is included without transformation in the chart output. Charts may therefore contain Kubernetes API objects